README.md

```diff
@@ -1,10 +1,86 @@
+
+If you have Node.js and `npm` already installed on your system, you can install a production frontend using the following command:
+
+```
+npm i carta-frontend
+```
+The files will be installed to your `node-modules` directory. For a system-wide 'global' installation (using the `-g` flag) it would be at `/usr/lib/node_modules`, whereas for a local user installation, it would be at `$PWD/node_modules`.
+
+You can install specific frontend versions by appending `@` plus the version number. For example,
+```
+npm i carta-frontend@1.4.0
+npm i carta-frontend@2.0.0-beta.0
+```
+The available versions are listed in the [npmjs.com carta-frontend versions tab](https://www.npmjs.com/package/carta-frontend?activeTab=versions).
+
+Alternatively, the pre-built production frontend can be downloaded directly without npm. For example,
+```
+wget https://registry.npmjs.org/carta-frontend/-/carta-frontend-2.0.0.tgz
+```
+The standalone `tgz` archive file extracts as `package/build` in your current directory.
+
+You can start the CARTA backend with a non-default frontend by specifying its location with the `--frontend_folder` flag. For example,
+```
+./carta-backend --frontend_folder $HOME/package/build
+```
+
+### PPA
+
+A Personal Package Archive (PPA) is a third-party package repository for Ubuntu. We supply pre-built production frontends at the [CARTAvis-team PPA on the Ubuntu Lauchpad](https://launchpad.net/~cartavis-team/+archive/ubuntu/carta).
+
+To install the latest release version:
+```
+sudo add-apt-repository ppa:cartavis-team/carta
+sudo apt-get update
+sudo apt-get install carta-frontend
+```
+
+The Ubuntu package install location for the frontend is `/usr/share/carta/frontend/`.
+
+(Once the PPA is added, you can also install the `carta` metapackage that includes both the CARTA backend and frontend components, as well as well as Beta versions of all the packages e.g. `carta-beta`, `carta-frontend-beta` and `carta-backend-beta`).
+
+### RPM
+
+RPMs are for users of Red Hat Enterprise Linux (RHEL) and other Red Hat-based Linux distributions, e.g. CentOS. 
+To install the latest release version, add the appropriate repository file depending on whether you are using RHEL7 (**el7**) or RHEL8 (**el8**):
+
+```
+sudo curl https://packages.cartavis.org/cartavis-el7.repo --output /etc/yum.repos.d/cartavis.repo
+```
+or
+```
+sudo curl https://packages.cartavis.org/cartavis-el8.repo --output /etc/yum.repos.d/cartavis.repo
+```
+Then the frontend component can be installed using `yum` or `dnf` :
+```
+sudo yum install carta-frontend
+```
+The RPM package install location for the frontend is `/usr/share/carta/frontend/`.
+
+(Once the RPM repo is added, you can also install the `carta` metapackage that includes both the CARTA backend and frontend components, as well as well as Beta versions of all the packages e.g. `carta-beta`, `carta-frontend-beta` and `carta-backend-beta`).
```

Comment:
Same suggestion as for the PPA:

"The RPM repo also provides a carta metapackage which automatically installs the CARTA frontend and backend packages. In addition to the latest release versions, beta versions are provided in a parallel set of packages."